controller/node_controller.go

```diff
@@ -309,7 +363,20 @@ func (nc *NodeController) syncNode(key string) (err error) {
 				}
 			}
 			node.Status.Conditions[types.NodeConditionTypeReady] = condition
+			break
+		}
+	}
+	// check physical node down
+	if removedNode {
```

Comment:
Should we monitor the Kubernetes nodes as well? If one node is in the Ready state but the manager is not running on it (if it's not in the process of starting up), there is also something wrong with it. That's more like the `KubernetesNodeDown` case.

What would happen if one node goes down? The manager pod would be removed from it?

Reply:
If we remove the kubernetes host from cluster, daemonset will remove the manager pod. But if node goes down such as power off, manager pod won't be removed and will report `NodeLost` or `Unknown`

Reply:
Ok, that makes sense. It's literally `node has been removed from Kubernetes` in this case then.

Reply:
Remember to test upgrade and uninstall/reinstall for this case. Make sure the status can be recovered as well.

Reply:
Do we also need to monitor kubernetes node to handle with node multiple status, such as NetworkUnavailable, DiskPressure, etc? If kubernetes node status is not ready, set longhorn node status to`Down` and show kubernetes node reason to longhorn node. 
In my investigation, if kubernetes node went down, it will take a very long time to update all pods status to `Unknown` or `NodeLost` on that down node. I think it's better for us to show longhorn node status as the time node went down instead of waiting for pod status.

Reply:
Yeah, we should do that. We can add another commit for it.